README.md

```diff
@@ -36,9 +36,15 @@ Similar to Task 1, start the data feed server by running the python server
 
 <code> python2 datafeed/server.py </code>
 
+If you encounter an issue with `datautil.parser`, run this command: 
+
+	pip install python-dateutil
+
+If you don't have pip, you can install it from: https://pip.pypa.io/en/stable/installing/
+
 Run <code>npm install</code> to start the React application.
 
-To run the app in development mode, run <code>npm start</code> in the project directory.
+To run the app in development mode, run <code>npm start</code> in the project directory. Make sure to run your terminal/command prompt as administrator.
```

Comment:
(indicate that running terminal / command prompt as admin only applies for windows)